Mathlib/CategoryTheory/Limits/Shapes/Biproducts.lean

```diff
@@ -640,6 +640,7 @@ lemma biproduct.whiskerEquiv_inv_eq_lift {f : J → C} {g : K → C} (e : J ≃
     · rintro rfl
       simp at h
 
+attribute [local simp] Sigma.forall in
```

Comment:
Can this be inlined into the `simp` calls in the proof?

Reply:
I was too slow, but: this was used by an autoparam `aesop` call.

Reply:
I guess I didn't read carefully